### PR TITLE
Fix a few typos within docs/

### PR DESCRIPTION
**Pull request by mohrash92**
Fixed a few typos in the documentation 😄

### CODE REVIEW

File: docs/signed-apk-android.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ You can create an APK for each CPU by changing the following line in android/app
...
 ```
 
-Upload both these files to markets which support device targetting, such as [Google Play](https://developer.android.com/google/play/publishing/multiple-apks.html) and [Amazon AppStore](https://developer.amazon.com/docs/app-submission/getting-started-with-device-targeting.html), and the users will automatically get the appropriate APK. If you want to upload to other markets, such as [APKFiles](https://www.apkfiles.com/), which do not support multiple APKs for a single app, change the following line as well to create the default universal APK with binaries for both CPUs.
+Upload both these files to markets which support device targeting, such as [Google Play](https://developer.android.com/google/play/publishing/multiple-apks.html) and [Amazon AppStore](https://developer.amazon.com/docs/app-submission/getting-started-with-device-targeting.html), and the users will automatically get the appropriate APK. If you want to upload to other markets, such as [APKFiles](https://www.apkfiles.com/), which do not support multiple APKs for a single app, change the following line as well to create the default universal APK with binaries for both CPUs.
```
Comment (rickhanlonii) [inline docs/signed-apk-android.md]: What changed here?

**Comment by kelset** [comment]
> @kelset which versions docs should we update this in to get it into the 0.60 release?

there should be a versioned_doc version called 0.60RC?

**Comment by rickhanlonii** [comment]
Are we cutting the next release docs from master? If so, we could wait until the next version, but if we're cherry-picking the docs then we should add it now?

**Comment by kelset** [comment]
I am not sure about that, @hramos will have a better answer, but I think the RC version will be "bumped" to become the 0.60 once we reach stable in terms of release.

**Comment by rickhanlonii** [comment]
Awesome, thanks! @mohrash92 do you want to add these updates to website/versioned_docs/version-0.60-RC as well so they go live with the next release?

File: docs/signed-apk-android.md (hunk 1 above)
Comment (mohrash92) [reply docs/signed-apk-android.md]: `targetting` changed to `targeting`. Not sure why it's not highlighting here but if you select the "Display the rich diff" button you'll be able to see it.

**Comment by mohrash92** [comment]
> Awesome, thanks! @mohrash92 do you want to add these updates to website/versioned_docs/version-0.60-RC as well so they go live with the next release?

Sure, thats done!

**Pull request merged**